services/src/main/java/org/keycloak/connections/httpclient/DefaultHttpClientFactory.java

```diff
@@ -134,8 +134,8 @@ private void lazyInit(KeycloakSession session) {
         if (httpClient == null) {
             synchronized(this) {
                 if (httpClient == null) {
-                    long socketTimeout = config.getLong("socket-timeout-millis", 5000L);
-                    long establishConnectionTimeout = config.getLong("establish-connection-timeout-millis", -1L);
+                    int socketTimeout = config.getInt("socket-timeout-millis", 5000);
```

Comment:
Why this change?

Reply:
Same as [this](https://github.com/keycloak/keycloak/pull/8165#discussion_r658516576)